server/src/main/java/com/objectcomputing/checkins/services/reviews/ReviewPeriod.java

```diff
@@ -35,9 +35,9 @@ public class ReviewPeriod {
     private String name;
 
     @NotNull
-    @Column(name = "open")
-    @Schema(description = "Whether or not the review period is open")
-    private Boolean open = true;
+    @Column(name = "status")
+    @Schema(description = "The current status of the review period")
+    private String status;
```

Comment:
Is there a reason to not use the `ReviewStatus` itself as the type of this column? I think this would simplify the code a little bit and also make the relationship of `ReviewPeriod` and `ReviewStatus` more clear.

Reply:
Nope, just from the initial passthrough. I've updated it to the use them Enum.